flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/CreditBasedPartitionRequestClientHandlerTest.java

```diff
@@ -419,6 +456,68 @@ public void testNotifyCreditAvailableAfterReleased() throws Exception {
 		}
 	}
 
+	@Test
+	public void testReceivedBufferForRemovedChannel() throws Exception {
+		final int bufferSize = 1024;
```

Comment:
nit: remove `final` to keep consistent in this test.

Reply:
Removed the final.